### DISCUSSION TITLE
Curso Fundamentos da Orientação a Objetos. Aula 'Modelando a carreira'

### DISCUSSION THREAD

**Discussion by pcostamiguez** [balta-io/forum#653] [Tira Dúvidas] [answered]
Boa tarde. Fazendo o curso me surgiu uma dúvida: Balta criou uma classe do tipo Enum para fazer referência aos níveis das aulas e dos cursos, contudo me pergunto se caso entrasse um tipo novo de Level ele teria que ser adicionado direto no código ou seria mais interessante manter o Level em um banco de dados, por exemplo? Grato.

**Comment by henriqueweege** [comment, marked as answer]
Oi, @pcostamiguez ! Tudo certo?
São abordagens diferentes que resolvem problemas diferentes. Para ter uma tabela no banco com essas infos, você precisaria ter todo o código para CRUD dessa tabela. Se parar para pensar, é bastante coisa para dar manutenção para algo que poderia ser um simples Enum. Se for o caso de precisar que alguém que consuma a aplicação crie esses níveis, faz sentido ter essa estrutura, caso contrário, não.

Valeeeeu!

**Comment by pcostamiguez** [comment]
Obrigado!
